torch/csrc/utils/python_arg_parser.h

```diff
@@ -345,6 +346,29 @@ inline c10::OptionalArray<int64_t> PythonArgs::intlistOptional(int i) {
   return intlist(i);
 }
 
+inline c10::OptionalArray<double> PythonArgs::doublelistOptional(int i) {
+  if (!args[i]) {
+    return {};
+  }
+  PyObject* arg = args[i];
+  // XXX no auto-broadcast
+  auto tuple = PyTuple_Check(arg);
+  auto size = tuple ? PyTuple_GET_SIZE(arg) : PyList_GET_SIZE(arg);
+  std::vector<double> res(size);
+  for (int idx = 0; idx < size; idx++) {
+    PyObject* obj = tuple ? PyTuple_GET_ITEM(arg, idx) : PyList_GET_ITEM(arg, idx);
+    try {
+      // XXX no variable check
```

Comment:
`intlist` checks for variables here.  Can we safely ignore that?

Reply:
that looks to be some hack that has to do tracing, I don't know whether we need to apply that same hack or whether that's even relevant anymore.  @eellison do you know?

Reply:
@jamesr66a said he would take a look